python/tests/display/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+# # This source code and the accompanying materials are made available under #
+# the terms of the Apache License 2.0 which accompanies this distribution. #
+# ============================================================================ #
+
+add_test(
```

Comment:
someone else can correct me if I'm wrong, but I'm not sure we need this CMakeLists file. The CI already globs up all  pytests in the `tests/*` folder so this new `test_draw.py` file should be run already. See here https://github.com/NVIDIA/cuda-quantum/blob/main/.github/workflows/test_in_devenv.yml#L209

Reply:
This makes sense. I had tried out a weird workaround for running this test apparently -- deleted!